static/app/components/events/contexts/device/getDeviceKnownDataDetails.tsx

```diff
@@ -26,22 +34,22 @@ export function getDeviceKnownDataDetails({
   type,
 }: Props): Output | undefined {
   switch (type) {
-    case DeviceKnownDataType.NAME:
+    case DeviceContextKeys.NAME:
```

Comment:
since this is an `enum` shall it be singular?

Reply:
fixed in b2503e906932b655b700dac5e6299e3ca1bd12a6